src/layouts/AssistenzaPage.jsx

```diff
@@ -0,0 +1,722 @@
+/* eslint-disable react/no-unescaped-entities */
+/* eslint-disable sonarjs/no-duplicate-string */
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable sonarjs/no-all-duplicated-branches */
+/* eslint-disable sonarjs/no-identical-functions */
+/* eslint-disable sonarjs/cognitive-complexity */
+/* eslint-disable max-lines-per-function */
+/* eslint-disable arrow-body-style */
```

Comment:
These ignore lines relax too much our linting checks, please consider to remove them.